language-server/modules/langserver-core/src/test/resources/completion/expression_context/source/list_constructor_ctx_source6.bal

```diff
@@ -0,0 +1,32 @@
+int[] arr = [1, 2, 3];
```

Comment:
Let's rename this file to show that we are doing `spead` related things.

Reply:
Done